src/ocamldep.mli

```diff
@@ -35,3 +39,9 @@ val rules_for_auxiliary_module
   :  Compilation_context.t
   -> Module.t
   -> Dep_graphs.t
+
+(** Get the dep graph for an already defined library *)
+val rules_for_lib
```

Comment:
Note that this doesn't produce rules, so it shouldn't be called `rules_...`. I suggest: `graph_of_remote_library`. In fact, maybe we should adopt the same kind of names for all these functions; the fact that they generate rules is not really interesting, it's just an implementation detail.